File: java/Class/Class/src/Class/Class/src/Class2.java
// 整数値と文字列を持つクラス
class Class2 {	// Class2クラスの定義

	// フィールドの定義
	private int i;	// int型フィールドi
	private String str;	// Stringクラスフィールドstr

	// メソッドの定義
	// 整数値と文字列をフィールドにセット.
	public void set(int n, String s) {	// メソッドsetの定義
		i = n;	// nをiにセット.
		str = s;	// sをstrにセット.
	}

	// 各フィールドを出力.
	public void output(){	// メソッドoutputの定義
		System.out.println("i = " + i + ", " + "str = " + str);	// iとstrを出力.
	}

}

File: cs/sealed/sealed/src/sealed/sealed/Class2.cs
// 名前空間の登録
using System;       // 共通データ型と基本クラス(System名前空間)

// Class2の定義
sealed class Class2 : Class1    // Class1を継承.
{
    // 継承対象の出力メソッド.
    public override void Print()
    {
        // "Class2.Print"と出力.
        Console.WriteLine("Class2.Print");    // Console.WriteLineで"Class2.Print"と出力.
    }
}